docs/machine-learning/how-to-guides/verify-model-quality-ml-net.md

```diff
@@ -7,6 +7,11 @@ ms.custom: mvc,how-to
 ---
```

Comment:
Can you can check this date as well?

Reply:
Done. Updated all the dates on all the How-To's.

https://github.com/dotnet/docs/pull/11020/commits/3afca9423998cf0449779598cd56af5e444d79ac